packages/twenty-front/src/i18n/config.ts

```diff
@@ -0,0 +1,48 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    fallbackLng: 'en',
+    debug: true,
+    interpolation: {
+      escapeValue: false,
+    },
+    resources: {
```

Comment:
In a future PR we should improve loading to lazy load efficiently and separate this into several files (but doesn't need to be part of this PR)